tests/integration/targets/elb_target/tasks/ec2_target.yml

```diff
@@ -296,6 +296,50 @@
           - result.proxy_protocol_v2_enabled == 'true'
           - result.preserve_client_ip_enabled == 'false'
 
+    # Modify target group atrributes =============================
+    # TODO: Add check_mode support
+
+    - name: update target group atrributes
+      elb_target_group:
+        name: "{{ tg_used_name }}"
+        health_check_port: 80
+        protocol: http
+        port: 80
+        vpc_id: "{{ vpc.vpc.id }}"
+        load_balancing_algorithm_type: least_outstanding_requests
+        state: present
+        target_type: instance
+        tags:
+          Description: "Created by {{ resource_prefix }}"
+      register: result
+
+    - name: update target group atrributes
+      assert:
+        that:
+          - result.changed
+          - result.target_group_name == tg_used_name
+          - result.load_balancing_algorithm_type == 'least_outstanding_requests'
+
+    - name: update target group atrributes - idempotence
+      elb_target_group:
+        name: "{{ tg_used_name }}"
+        health_check_port: 80
+        protocol: http
+        port: 80
+        vpc_id: "{{ vpc.vpc.id }}"
+        load_balancing_algorithm_type: least_outstanding_requests
+        state: present
+        target_type: instance
+        tags:
+          Description: "Created by {{ resource_prefix }}"
+      register: result
+
+    - name: update target group atrributes - idempotence
+      assert:
+        that:
+          - not result.changed
+          - result.load_balancing_algorithm_type == 'least_outstanding_requests'
```

Comment:
Can you also verify for 'round_robin' on an existing test (_verify the default_)?

Otherwise LGTM

Reply:
@markuman just to confirm I added to an existing test and can confirm the default applies correctly. the type is only returned for elb type target groups, for the tcp nlb ones its not returned at all by AWS